graphs/adapter-script-generation/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/scriptgen/UndirectedGraphScriptGenActionFactory.kt

```diff
@@ -0,0 +1,117 @@
+package com.anaplan.engineering.azuki.graphs.adapter.scriptgen
+
+import com.anaplan.engineering.azuki.core.system.Action
+import com.anaplan.engineering.azuki.core.system.ParallelAction
+import com.anaplan.engineering.azuki.graphs.adapter.api.DirectedActionFactory
+import com.anaplan.engineering.azuki.graphs.adapter.api.UndirectedActionFactory
+import com.anaplan.engineering.azuki.graphs.adapter.api.UndirectedGraphActionFactory
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.action.AddEdgeToDirectedGraphDeclarableAction
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.action.AddEdgeToUndirectedGraphDeclarableAction
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.action.AddVertexToDirectedGraphDeclarableAction
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.action.AddVertexToUndirectedGraphDeclarableAction
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.action.CreateUndirectedGraphDeclarableAction
+import com.anaplan.engineering.azuki.graphs.dsl.action.DirectedGraphActions
+import com.anaplan.engineering.azuki.graphs.dsl.action.UndirectedGraphActions
+import com.anaplan.engineering.azuki.script.generation.ScriptGenerationAction
+
+object UndirectedGraphScriptGenActionFactory : UndirectedGraphActionFactory<ScriptGenerationAction> {
```

Comment:
🚫 Same as for the others, this shouldn't have changed.
```suggestion
object GraphScriptGenActionFactory : GraphActionFactory<ScriptGenerationAction> {
```